addons/csw/stringtable.xml

```diff
@@ -658,6 +688,18 @@
             <Russian>[CSW] Лента 20-мм гранат для ст. гранатомёта</Russian>
             <Korean>[CSW] 20mm 고속유탄발사기 탄띠</Korean>
         </Key>
+        <Key ID="STR_ACE_CSW_GMGBelt96_displayName">
+            <English>[CSW] 40mm Grenade GMG Belt 96 Count</English>
+        </Key>
+        <Key ID="STR_ACE_CSW_SPGHEAT_displayName">
+            <English>[CSW] SPG9 HEAT Rounds</English>
+        </Key>
+        <Key ID="STR_ACE_CSW_SPGHE_displayName">
+            <English>[CSW] SPG9 HE Rounds</English>
+        </Key>
+        <Key ID="STR_ACE_CSW_65_39Belt_displayName">
+            <English>[CSW] 6.5X39mm Belt</English>
```

Comment:
```suggestion
            <English>[CSW] 6.5x39 mm Belt</English>
```

BI mags add a space before the 'mm'. We should do the same for consistency. The 'X' is never capitalized.